packages/cw0/src/balance.rs

```diff
@@ -65,7 +65,7 @@ impl Balance {
     }
 
     pub fn is_empty(&self) -> bool {
-        self.0.is_empty()
+        !self.0.iter().any(|x| x.amount != Uint128(0))
```

Comment:
Thanks for updating this

Reply:
You're welcome. That's a clever check by the way.